### PR TITLE
Box compontent

### PR DESCRIPTION
**Pull request by woutsluiter**
### This PR:

**Adds** ✨
- `Box` component for aligning child components

### CODE REVIEW

File: src/components/Row/Row.template.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { StatelessComponent } from 'react';
+
+type PropsType = {
```
Comment (CarloPalinckx) [inline src/components/Row/Row.template.tsx]: you can intersect `StyledType` and then you can also drop `className`.

File: src/components/Row/Row.test.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import Row from '.';
+
+describe('Row', () => {
+    it('aligns children to the right', () => {
+        const row = shallow(<Row horizontal="right" />);
```
Comment (CarloPalinckx) [inline src/components/Row/Row.test.tsx]: You need to wrap enzyme's shallowWrapper in a `toJson` imported from `enzyme-to-jest` to get a proper snapshot.

**Comment by CarloPalinckx** [comment]
So you we're talking about adding a `Column` component to this to deal with horizontal spacing right? How are you planning to handle the "flex children" ?

**Comment by woutsluiter** [comment]
But now that i'm looking at it, I wonder if a separate element is necessary. Maybe a single `Flex` component would suffice with the ability to flex itself (basis, grow, shrink)

**Comment by woutsluiter** [comment]
But then again, this notation seems really self explanatory 🤔

**Pull request closed without merging**